docs/tutorials/constraints.rst

```diff
@@ -0,0 +1,255 @@
+
+.. code-block:: python
+
+    from astroplan import (AltitudeConstraint, AirmassConstraint,
+                           AtNightConstraint)
+    constraints = [AltitudeConstraint(10*u.deg, 80*u.deg),
+                   AirmassConstraint(5), AtNightConstraint.twilight_civil()]
+
+This list of constraints can now be applied to our target list to determine
+whether:
+
+* the targets are observable given the constraints at *any* times in the time
+  range, using `astroplan.is_observable`,
+
+* the targets are observable given the constraints at *all* times in the time
+  range, using `astroplan.is_always_observable`::
+
+    from astroplan import is_observable, is_always_observable
+    # Are targets *ever* observable in the time range?
+    ever_observable = is_observable(constraints, subaru, targets, time_range=time_range)
+
+    # Are targets *always* observable in the time range?
+    always_observable = is_always_observable(constraints, subaru, targets, time_range=time_range)
+
+These two functions will return boolean arrays which tell you whether or not
+each target is observable given your constraints. Let's print these results in
+tabular form:
+
+    >>> from astropy.table import Table
+    >>> import numpy as np
+    >>> observability_table = Table()
+    >>> observability_table['targets'] = [target.name for target in targets]
+    >>> observability_table['ever_observable'] = ever_observable
+    >>> observability_table['always_observable'] = always_observable
+    >>> print(observability_table)
+    <Table length=6>
+    targets ever_observable always_observable
+      str7        bool             bool
+    ------- --------------- -----------------
+    Polaris            True              True
+       Vega            True              True
+    Albireo            True             False
+      Algol            True             False
+      Rigel           False             False
+    Regulus           False             False
+
+Now we can see which targets are observable! You can also use the
+`astroplan.observability_table` method to do the same calculations and store the
+results in a table, all in one step::
+
+    >>> from astroplan import observability_table
+    >>> table = observability_table(constraints, subaru, targets, time_range=time_range)
+    >>> print(table)
+    target name ever observable always observable fraction of time observable
+    ----------- --------------- ----------------- ---------------------------
+        Polaris            True              True                         1.0
+           Vega            True              True                         1.0
+        Albireo            True             False              0.833333333333
+          Algol            True             False              0.166666666667
+          Rigel           False             False                         0.0
+        Regulus           False             False                         0.0
+
```

Comment:
Can you add in the example from the ipython notebook you sent around which actually makes a plot showing points for when things are observable?  Or was that too complicated? (Ideally you should just be able to use `plot_sky`, and point users to the plotting tutorial section for more on that)